extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -1030,6 +1052,7 @@ export class SKURecommendationPage extends MigrationWizardPage {
 
 				else if (this.migrationStateModel.performanceCollectionStopped()) {
 					this._skuDataCollectionStatusText.value = constants.AZURE_RECOMMENDATION_STATUS_STOPPED;
+					this._skuDataCollectionTimerText.value = constants.EMPTY;
```

Comment:
we can leave this for now, but I will probably update this so that the _skuDataCollectionTimerText gets hidden when perf collection is stopped